src/events/EventManager.ts

```diff
@@ -1,133 +1,143 @@
-import { EmitterSubscription, NativeEventEmitter, NativeModule } from 'react-native';
+import {
+  EmitterSubscription,
+  NativeEventEmitter,
+  NativeModule,
+} from 'react-native';
 import NotificationReceivedEvent from './NotificationReceivedEvent';
-import { isMultipleInstancesPossible } from '../helpers';
+import {isMultipleInstancesPossible} from '../helpers';
```

Comment:
there should be spaces between bracket and name. probably a rule that needs to be added.

for consistency with other SDKs

Reply:
I've pushed a commit with bracketSpacing enabled.  Let me know what you think.